app/pdfs/distribution_pdf.rb

```diff
@@ -110,5 +112,72 @@ def initialize(organization, distribution)
         end
       end
     end
+
+    render
+  end
+
+  def request_data
+    data = [["Items Received",
+      "Requested",
+      "Received",
+      "Value/item",
+      "In-Kind Value Received",
+      "Packages"]]
+
+    request_items = @distribution.request.request_items.dup
+    line_items = @distribution.line_items.sorted
+
+    requested_not_received = request_items.filter_map do |request_item|
+      if line_items.none? { |i| i.item_id == request_item["item_id"] }
+        RequestItem.from_json(request_item, @distribution.request)
+      end
+    end
+
+    data += line_items.map do |c|
+      [c.item.name,
+        c.quantity,
+        c.quantity,
+        dollar_value(c.item.value_in_cents),
+        dollar_value(c.value_per_line_item),
+        c.package_count]
+    end
+
+    data += requested_not_received.sort_by(&:name).map do |c|
+      [c.item.name,
+        c.quantity,
+        "",
+        dollar_value(c.item.value_in_cents),
+        dollar_value(c.value_per_line_item),
+        c.package_count]
+    end
```

Comment:
Think there is something not working in this logic. I tested this out locally and it looks like the requested & the distributed always match. Try requesting a request as a partner and fulling the distribution with different quantity. It looks like the requested doesn't match the original requested. For example, the requested amount was 10 not 5 in this snapshot:
![Captura de Pantalla 2022-10-15 a la(s) 5 16 21 p m](https://user-images.githubusercontent.com/11335191/196009620-8e110076-21e1-472a-8f8b-d3a657a7ea7e.png)